--- ./src/main/java/com/google/devtools/build/lib/buildeventstream/proto/BUILD
+++ ./src/main/java/com/google/devtools/build/lib/buildeventstream/proto/BUILD
@@ -25,6 +25,7 @@ java_proto_library(
 proto_library(
     name = "build_event_stream_proto",
     srcs = ["build_event_stream.proto"],
+    visibility = ["//visibility:public"],
     deps = [
         "//src/main/protobuf:command_line_proto",
         "//src/main/protobuf:failure_details_proto",
